packages/shared/src/components/cards/SharedPostText.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement, useLayoutEffect, useRef } from 'react';
+
+type SharedPostTextProps = {
+  title: string;
+  onHeightChange?: (height: number) => void;
+};
+
+export const SharedPostText = ({
+  title,
+  onHeightChange,
+}: SharedPostTextProps): ReactElement => {
+  const sharedPostTitleRef = useRef<HTMLDivElement>();
+  useLayoutEffect(() => {
```

Comment:
Could potentially be heavy calculation on long feeds, will need to test it doesn't impact scrolling. (and responsive changing the feed)

Reply:
It doesnt seem to have any effect on my tests.

Reply:
Let's keep this in mind in case we see production issues